src/bumerang/borrowsbyrecenthandler.py

```diff
@@ -0,0 +1,30 @@
+from bumerang.bumerangrequesthandler import BumerangRequestHandler
+from bumerang.error import BumerangError
+from json import dumps
+
+class BorrowsByRecentHandler(BumerangRequestHandler):
+    """ This class handles fetching multiple bumerang requests depending
+        on which are most recently created.
+    """
+
+    def get(self, num_requests=10):
+        """Obtain a list of requests."""
+
+        try:
+            num = int(num_requests)
```

Comment:
Does this work as the basic type it is from the request? It's thrown into the sql which is all a string